src/globus_sdk/_testing/models.py

```diff
@@ -40,14 +59,44 @@ class RegisteredResponse:
     def __init__(
         self,
         *,
+        # path and service are glbous-sdk specific
+        # in `responses`, these are just `url`
         path: str,
-        service: str | None = None,
-        method: str = responses.GET,
+        service: (
+            Literal[
```

Comment:
Looks like we have a pyupgrade configuration issue: above, `Literal` is imported conditionally for Python 3.7 and below, which we no longer support.

This should become `t.Literal` in a subsequent PR.